src/app/(with-sidebar)/myfeed/types/types.ts

```diff
@@ -107,8 +86,8 @@ export type PostCardContentProps = {
 
 export type FollowButtonProps = {
   authorProfileLink: string;
-  yPadding?: string;
-  xPadding?: string;
+  style?: SxProps;
+  followState?: string;
```

Comment:
can we use boolean instead of string?

Reply:
followState is like the text initalization for followbutton so followstate default is follow, which serves as text for the button and the text change with the follow state